apollo-api/api/apollo-api.api

```diff
@@ -101,6 +101,12 @@ public final class com/apollographql/apollo3/api/ApolloResponse$Builder {
 	public final fun requestUuid (Ljava/util/UUID;)Lcom/apollographql/apollo3/api/ApolloResponse$Builder;
 }
 
+public final class com/apollographql/apollo3/api/ApolloResponsesKt {
```

Comment:
Use `@JvmName("ApolloResponses")` to make Java interop nicer?

Reply:
Also what about merging with `ResponseParser` and have something like `ApolloResponseAdapter` that can adapt to//from json like we do for other models?